securing_apps/topics/oidc/java/params_forwarding.adoc

```diff
@@ -30,3 +34,7 @@ The supported parameters are:
 Most of the parameters are described in the http://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint[OIDC specification].
 The only exception is parameter `kc_idp_hint`, which is specific to {project_name} and contains the name of the identity provider to automatically use.
 For more information see the `Identity Brokering` section in link:{adminguide_link}[{adminguide_name}].
+
+WARNING: Currently if you open the URL with the attached parameters, the adapter won't redirect you to the {project_name} in case that you
+are already authenticated in the application. For example opening `$$http://myappserver/mysecuredapp?prompt=login$$` won't automatically redirect you to the {project_name}
+login page in case that you are already authenticated to the application `mysecredapp` . This behaviour can be changed in the future.
```

Comment:
WARNING: If you open the URL using the attached parameters, the adapter will not redirect you to {project_name} if you are already authenticated in the application. For example, opening `$$http://myappserver/mysecuredapp?prompt=login$$` will not automatically redirect you to the {project_name} login page if you are already authenticated to the application `mysecredapp` . This behavior may be changed in the future.